src/main.cpp

```diff
@@ -3,6 +3,7 @@
 
 #include <spdlog/spdlog.h>
 #include <docopt/docopt.h>
+#include <include/version.hpp>
```

Comment:
```suggestion
#include "include/version.hpp"
```

`<>` should be reserved for system includes according to some style guides/static analyzers